databricks/koalas/indexes.py

```diff
@@ -422,6 +460,15 @@ def to_pandas(self) -> pd.MultiIndex:
 
     toPandas = to_pandas
 
+    # TODO: add 'name' parameter after pd.MultiIndex.name is implemented
+    def copy(self):
+        """
+        Make a copy of this object.
+        """
```

Comment:
Maybe we should add an example but I think it's fine for now since it's incomplete for now due to missing name anyway.

Reply:
thanks for reviewing. i'm planning to implement `name`, after then i'll add the example here also.